### PR TITLE
Ch13rev

### PR DESCRIPTION
**Pull request by Nowosad**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Is this ready for review @Nowosad ? Thanks for the review!

File: 13-transport.Rmd [hunk 1]
```diff
@@ -72,17 +73,15 @@ An overview of the region's transport network is illustrated in Figure \@ref(fig
...
+# summary(bristol_ttwa)
+# summary(bristol_region)
 region_all = rbind(bristol_region, bristol_ttwa)
 tmap_mode("view")
 tm_shape(region_all[1, ], bbox = region_all) +
   tm_fill("yellow", alpha = 0.5) +
   tm_shape(bristol_ways) +
-  tm_lines(col = "highway", lwd = 2.1, palette = "-Set1"
-    ) +
+  tm_lines(col = "highway", lwd = 2.1, palette = "-Set1") +
```
Comment (Robinlovelace) [inline 13-transport.Rmd:84]: :+1:

File: 13-transport.Rmd [hunk 2]
```diff
@@ -294,7 +291,7 @@ bristol_od$Active = (bristol_od$bicycle + bristol_od$foot) /
   bristol_od$all * 100
 ```
 
-There are two main types of OD pair: *interzonal* and *intrazonal*.
+There are two main types of OD pairs: *interzonal* and *intrazonal*.
```
Comment (Robinlovelace) [inline 13-transport.Rmd:294]: Initially I thought this change was wrong, but now think you're right: there are OD pairs come in many types. https://english.stackexchange.com/questions/29843/types-of-followed-by-singular-or-plural

File: 13-transport.Rmd [hunk 3]
```diff
@@ -437,9 +435,9 @@ Routes are generated from desire lines or, more commonly, matrices containing co
 This routing process is done by a range of broadly-defined *routing engines*: software and web services that return geometries and attributes describing how to get from origins to destinations.
 Routing engines can be classified based on *where* they run relative to R:
 
--   In-memory routing using R packages that enable route calculation (described in Section \@ref(memengine))
--   Locally hosted routing engines external to R that can be called from R (Section \@ref(localengine))
--   Remotely hosted routing engines by external entities that provide a web API that can be called from R (\@ref(remoteengine))
+- In-memory routing using R packages that enable route calculation (described in Section \@ref(memengine))
+- Locally hosted routing engines external to R that can be called from R (Section \@ref(localengine))
+- Remotely hosted routing engines by external entities that provide a web API that can be called from R (Section \@ref(remoteengine))
```
Comment (Robinlovelace) [inline 13-transport.Rmd:440]: :+1:

File: 13-transport.Rmd [hunk 4]
```diff
@@ -613,15 +612,14 @@ route_network_scenario = overline(routes_short_scenario, attrib = "bicycle")
...
 routes_short_scenario |> 
   ggplot() +
   geom_line(aes(distance / 1000, uptake)) +
-  xlab("Route distance (km)") +
-  ylab("Percent trips switching from driving to cycling") +
+  labs(x = "Route distance (km)", y = NULL, title = "Percent trips switching from driving to cycling") +
   scale_y_continuous(labels = scales::percent)
 tm_shape(route_network_scenario) +
-  tm_lines(lwd = "bicycle", scale = 9, title.lwd = "N. bike trips per day\n(modeled, one direction)")
+  tm_lines(lwd = "bicycle", scale = 9, title.lwd = "No. bike trips per day\n(modeled, one direction)")
```
Comment (Robinlovelace) [inline 13-transport.Rmd:622]: I vote Number of bike trips, to be explicit.

File: 13-transport.Rmd [hunk 5]
```diff
@@ -692,17 +690,16 @@ The methods presented in this section are relatively simple compared with what i
...
 A final point is that the example dataset we used above is relatively small.
-It may also be worth considering how the work could adapt to larger networks: testing methods on a subset of the data, and ensuring you have enough RAM will help, although it's also worth exploring other tools that can do transport network analysis that are optimised for large networks, such as R5 [@alessandretti_multimodal_2022].
-
+It may also be worth considering how the work could adapt to larger networks: testing methods on a subset of the data, and ensuring you have enough RAM will help, although it's also worth exploring other tools that can do transport network analysis that are optimized for large networks, such as R5 [@alessandretti_multimodal_2022].
 
 ## Prioritizing new infrastructure
 
-This final practical section demonstrates the policy relevance of geocomputation for transport planning and policies, with a simple example to help prioritise locations new investment.
+This final practical section demonstrates the policy relevance of geocomputation for transport planning and policies, with a simple example to help prioritize locations of a new investment.
```
Comment (Robinlovelace) [inline 13-transport.Rmd:697]: I don't think "a " is needed here: there is usually more than one.

File: 13-transport.Rmd [hunk 6]
```diff
@@ -692,17 +690,16 @@ The methods presented in this section are relatively simple compared with what i
...
 Clearly, the approach is deliberately simplified for educational purposes.
 The analysis would need to be substantially expanded --- including with larger input datasets --- to inform transport planning design, investment and policy decisions (possible next steps are discussed in Section \@ref(future-directions-of-travel)).
 
 An advantage of the data driven approach outlined in this chapter is its modularity: each aspect can be useful on its own, and feed into wider analyses.
-The steps that got us to this stage included identifying short but car-dependent commuting routes (generated from desire lines) in Section \@ref(routes) and analsysi of route network characteristics with the **sfnetworks** package in Section \@ref(route-networks).
+The steps that got us to this stage included identifying short but car-dependent commuting routes (generated from desire lines) in Section \@ref(routes) and analysis of route network characteristics with the **sfnetworks** package in Section \@ref(route-networks).
```
Comment (Robinlovelace) [inline 13-transport.Rmd:702]: Woops.

File: _13-ex.Rmd [hunk 7]
```diff
@@ -53,10 +52,13 @@ sum(d_intersect$all) / sum(desire_lines$all) # 88%
 ```
 
 E4. The analysis presented in this chapter is designed for teaching how geocomputation methods can be applied to transport research. If you were doing this for real, in government or for a transport consultancy, what top 3 things would you do differently?
-<!-- Higher level of geographic resolution. -->
-<!-- Use cycle-specific routing services. -->
-<!-- Identify key walking routes. -->
-<!-- Include a higher proportion of trips in the analysis -->
+
+```{r}
```
Comment (Robinlovelace) [inline _13-ex.Rmd:56]: Why this?

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
I made a few comments. Overall this is good and I'm happy for it to be merged. Can always do post-post-merge review PR!

File: 13-transport.Rmd [hunk 8]
```diff
@@ -1,10 +1,5 @@
 # (PART) Applications {.unnumbered}
 
-```{r, echo=FALSE}
-knitr::opts_chunk$set(warning = FALSE)
```
[the remark this answers is not in the document]
Comment (Nowosad) [reply 13-transport.Rmd:4]: This chunk was duplicated. Its second copy is still here.

File: 13-transport.Rmd (hunk 4 above)
Comment (Nowosad) [reply 13-transport.Rmd:622]: No. is fine by me (better than N.); If you prefer "Number" you need to change it in several figures in this chapter.

File: 13-transport.Rmd (hunk 5 above)
Comment (Nowosad) [reply 13-transport.Rmd:697]: So "new investments"?

File: _13-ex.Rmd (hunk 7 above)
Comment (Nowosad) [reply _13-ex.Rmd:56]: To render it on the solution website. If it is an html comment, it is invisible there. However, if it is an R code comment -- you can see it on the solution website.

**Comment by Nowosad** [comment]
@Robinlovelace sorry, I forgot to "undraft" this PR yesterday.

File: 13-transport.Rmd (hunk 5 above)
Comment (Robinlovelace) [reply 13-transport.Rmd:697]: No, investment that is spent in many places. E.g.: 

> The UK government has announced billions of pounds of new investment in roads and charging infrastructure for electric vehicles as part of its Spending Review 2020.

That is better than "new investments in roads" and the same applies to locations of new investment. There's a definite typo fix with the addition of "of" so let's merge it. Happy to make post merge changes.

Source: https://www.eunomia.co.uk/category/policy-strategy/sustainable-future-transport/

File: _13-ex.Rmd (hunk 7 above)
Comment (Robinlovelace) [reply _13-ex.Rmd:56]: Nice! Forgot that detail.

File: 13-transport.Rmd (hunk 5 above)
Comment (Nowosad) [reply 13-transport.Rmd:697]: 👍🏻

**Pull request merged**